bluemira/power_cycle/net/loads.py

```diff
@@ -131,32 +105,22 @@ def _shift_time(self, time_shift):
         Stores the shifting factor in the attribute '_shift', which
         is always initialized as an empty list.
         """
-        time_shift = validate_numerical(time_shift)
-        time = self.time
-        shifted_time = [t + time_shift for t in time]
-        self.time = shifted_time
+        self.time = time_shift + self.time
         self._shift.append(time_shift)
 
     def make_consumption_explicit(self):
```

Comment:
Upon revision, all load values should be inverted, so let's just put here:
```
self.data = -self.data
```
My concern is that if we call this function twice, it will invert values again. So the class should have a flag to indicate whether inversion has already happened.